gtsam/nonlinear/ISAM2.cpp

```diff
@@ -773,14 +796,22 @@ ISAM2Result ISAM2::update(
     variableIndex_.augment(newFactors, result.newFactorsIndices);
   else
     variableIndex_.augment(newFactors);
+
+  // Augment it with existing factors which now affect to more variables:
+  if (up.newAffectedKeys) {
+    for (const auto &fk : *up.newAffectedKeys) {
```

Comment:
fk ? In general, don't abbreviate. I know we used to do this, but I frown upon it now :-)